app/services/group_stage_service.rb

```diff
@@ -55,5 +55,35 @@ def update_group_scores(group)
       end
       changed_group_scores
     end
+
+    # Returns a list of the teams in the group sorted by their group_points, difference_in_points, scored_points
+    #
+    # @param group Group the group to get the teams from
+    # @return [Array] of teams
+    def teams_sorted_by_group_scores(group)
+      group.teams.sort do |a, b|
+        [group.group_scores.find_by(team: b).group_points,
+         group.group_scores.find_by(team: b).difference_in_points,
+         group.group_scores.find_by(team: b).scored_points] <=>
```

Comment:
please assign the return value of `...find_by()` to a var, they cause a db query every time...

Reply:
done